### ISSUE TITLE
I cannot play lan multiplayer

### ISSUE THREAD

**Issue by Deson09** [zhuowei/MCPELauncher#424]
Can you guys fix where I join a lan multiplayer and it crashes? Really wanna play with my friends with mods.

**Comment by KairusDarkSeeker** [comment]
What version of mcpe are you using?

**Comment by Deson09** [comment]
0.13.1

**Comment by sero583** [comment]
Yeah i already made the same Issue, well but most doesn't used their brains and wrote anything what they wanted to do.
